### PR TITLE
form-control class added to select box

### PR DESCRIPTION
**Pull request by Thirumal-Sakthivel**
This is existing select box UI,
existing_select

I have added  class to select box in Profile Notification page.
new_select

Form group alignment issue fixed in <strong>webhooks index</strong> page.

Alignment issue screenshot,

existing_align

Alignment issue fixed screenshot,

fixed_align

Please verify and merge my commit.

### CODE REVIEW

File: app/views/projects/hooks/index.html.haml [hunk 1]
```diff
@@ -19,7 +19,7 @@
       = f.text_field :url, class: "form-control", placeholder: 'http://example.com/trigger-ci.json'
   .form-group
     = f.label :url, "Trigger", class: 'control-label'
-    .col-sm-10
+    .col-sm-10.form-control-padding-top
```
Comment (randx) [inline app/views/projects/hooks/index.html.haml]: Will it work with `prepend-top-10` css class?

Comment (Thirumal-Sakthivel) [inline app/views/projects/hooks/index.html.haml]: Yes @randx it will work.

Comment (randx) [inline app/views/projects/hooks/index.html.haml]: @Thirumal-Sakthivel cool. Lets use it because its already defined

Comment (Thirumal-Sakthivel) [inline app/views/projects/hooks/index.html.haml]: @randx I used <code>prepend-top-10</code> class  and pushed.

**Comment by randx** [comment]
Thanks

**Comment by dosire** [comment]
@Thirumal-Sakthivel Congrats on the recognition https://twitter.com/spritlesoftware/status/664741967583023104

**Pull request merged**